### DISCUSSION TITLE
Recommended CS-Pin for Display Type 1 (PDC8544)

### DISCUSSION THREAD

**Discussion by huckes** [tbnobody/OpenDTU#612] [Q&A]
Is there any recommendation available for the CS-Pin for the display type 1? Or, even more conveniently a json-File for the PIN-Mapping?

The PCD8444 Display is a SPI-driven Device, so MISO/MOSI and SCLK are assumed to be the same as for the NRF-Module.
But CS must be connected to another GPIO.

**Comment by dAjaY85** [comment]
Hi,
not yet.
It may work with the HSPI, but I must test it first.
I Ordered one Nokia, for this test.
